docs/mcd.md

```diff
@@ -182,8 +225,6 @@ The images show the Connection Dialog produced using the Connection Fields file
     </conditions>
   </field>
  
-  <field name="AutoReconnect" label="" value-type="string" default-value="0" editable="false" />
```

Comment:
This was intentional removal

Reply:
It would be great if you can add sslmode in the example.

Reply:
I'll add a note to do as follow-up in the TFS item.  I'll need to update the screenshots if I change examples here.

Reply:
Instead of modifying existing example I opted to make a clean boolean focused Example 3.